07/28/2024 09:52:13 AM Logging starts
Traceback (most recent call last):
  File "./popular_projects_snippets_dataset/flask/bodies/body_320_guide_3_6.py", line 31, in <module>
    raise ValueError("Resources can only be opened for reading.")
ValueError: Resources can only be opened for reading.
Command exited with non-zero status 1
1.56user 4.73system 0:00.69elapsed 902%CPU (0avgtext+0avgdata 107948maxresident)k
0inputs+24outputs (0major+25065minor)pagefaults 0swaps
